README.md

```diff
@@ -40,6 +40,24 @@ We are considering using a tool like [Next.js](https://nextjs.org) to get up and
 
 We will focus on building an open API first. Then developers can use the API to build their own mobile clients and voice interface clients.
 
+## Development Setup
+
+Requirements: Docker, internet access
+
+Install dependencies:
+
+```
+npm install
```

Comment:
Why is `npm` a dependency, doesn't everything run in Docker?

_(Sorry, don't know much about JavaScript stuff.)_

Reply:
Good catch!  Using a dockerfile here would actually make it cleaner, since running `npm install` should be part of the build step.  Will make a simple dockerfile for this.